pillar/nginx/micromasters_es.sls

```diff
@@ -25,7 +25,7 @@ nginx:
                 - listen:
                     - '[::]:443'
                     - ssl
-                - location ~ ^/(_alias|_aliases|micromasters|_refresh|_mapping):
+                - location ~ ^/(_alias|_aliases|micromasters|_refresh|_mapping|_search):
```

Comment:
Can we limit this to the scroll API, at `_search/scroll`?

Reply:
My understanding from the docs is that the scroll API uses a query parameter and not an actual route, but I could very well be wrong about that.

Reply:
I mean, instead of `_search` we should do `_search/scroll` since all other `_search` calls use the index in the URL